manifests/kustomize/base/metadata/metadata-deployment.yaml

```diff
@@ -20,15 +20,16 @@ spec:
         env:
         - name: GRPC_PORT
           value: "8080"
-        - name: METADATA_STORE_SERVER_CONFIG_FILE
-          value: "/etc/config/mlmd_config.prototxt"
+        - name: MY_SQL_PASSWORD
```

Comment:
TODO: This needs to be updated to use right environment variable.

Reply:
Updated based on the secret values used in https://github.com/kubeflow/pipelines/pull/2067